docs/marketplace/add_to_marketplace/contributing_overview.md

```diff
@@ -21,8 +21,17 @@ To achieve this, a Company Owner or a Project Administrator can create new Compa
 Company Owners and Project Administrators can create, delete and update those Markeplace items exclusively available for their Company, but they cannot manage public Marketplace items.  
 The Console Super User role, instead, has permission to manage all Marketplace items from the CMS without any limitations.
 
+### Deciding whether to create a private or public marketplace item
+
+The choice between a private and public marketplace involves benefits on both sides that should be carefully analyzed before making a decision.
+
+Creating an item within the private marketplace is undoubtedly the better choice when you want to make something highly specialized available for the specific use cases of your company. If the item cannot be generalized and would have little value for other tenants, then a private marketplace is the way to go. Additionally, in this case, creating items within the private marketplace allows for complete ownership in managing your plugin, resulting in a faster maintenance process.
```

Comment:
```suggestion
Creating an item within the private marketplace is undoubtedly the better choice when you want to make something highly specialized available for the specific use cases of your Company. If the item cannot be generalized and would have little value for other Companies, then a private marketplace is the way to go. Additionally, in this case, creating items within the private marketplace allows for complete ownership in managing your plugin, resulting in a faster maintenance process.
```

we don't talk about tenants anymore, and if you are referring to the Company resource within the console it's better to use the first letter capitalized